demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -346,7 +346,7 @@ def upload(
         marketplace: MarketplaceVersions,
         target_demisto_version: Version,
         destination_zip_dir: Optional[Path] = None,
-        zip: bool = False,
+        zip: bool = True,
```

Comment:
it doesn't break current functionality in any way?

Reply:
No, as the default is actually `True` now in the upload command.